local_costmap:
  global_frame: odom
  robot_base_frame: base_footprint

  update_frequency: 10
  publish_frequency: 10
  transform_tolerance: 1

  static_map: false  
  rolling_window: true
  width: 4
  height: 4
  resolution: 0.01
  
